examples/sample_prescriptions_task.csv

```diff
@@ -0,0 +1,3 @@
+StartDate,EndDate,IpFile,CostFile,OutputFile
+2020-08-16,2020-08-30,2020-09-30_historical_ip.csv,ip_costs.csv,predictions/2020-08-16_2020-08-12.csv
```

Comment:
The OutputFile name should be something different: it should go to `prescriptions/`, and if the name means something (like stardate_enddate.csv) it should be `2020-08-16_2020-08-30.csv` or even `2020-08-16_2020-08-30_task1.csv`